### PR TITLE
Run WSIViewer from `runpp()`

### PR DESCRIPTION
**Pull request by creilly8**
## Description
Addresses the first task in issue #2021. 

Changes: 
1. WSIviewer launches as a plot type from `runpp()`

Test: 
1. Remove `.skip` in`/client/plots/wsiviewer/test/wsiviewer.integration.spec.ts`

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/plots/wsiviewer/defaults.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import Settings from './Settings'
-import { copyMerge } from '#rx'
+import { copyMerge } from '../../rx'
```
Comment (aacic) [inline client/plots/wsiviewer/defaults.ts:2]: Please revert this like so it's:
`import { copyMerge } from '#rx'`

File: client/plots/wsiviewer/WSIViewer.ts [hunk 2]
```diff
@@ -13,6 +13,8 @@ import { WSIViewerInteractions } from '#plots/wsiviewer/interactions/WSIViewerIn
 import Settings from '#plots/wsiviewer/Settings.ts'
 import wsiViewerDefaults from '#plots/wsiviewer/defaults.ts'
 import { GetWSImagesRequest, GetWSImagesResponse } from '../../shared/types/routes/wsimages.ts'
+import wsiViewerImageFiles from '#plots/wsiviewer/images.ts'
```
Comment (xzhou82) [inline client/plots/wsiviewer/WSIViewer.ts:16]: import from `./images.ts` instead?

File: client/plots/wsiviewer/test/wsiviewer.integration.spec.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+	},
+	debug: 1
+})
+
+tape.skip('\n', function (test) {
+	test.pass('-***- plots/wsiviewer/wsiviewer -***-')
+	test.end()
+})
+
+//Data not available to run in Termdb test (i.e. CI)
```
Comment (aacic) [inline client/plots/wsiviewer/test/wsiviewer.integration.spec.ts:27]: In order to make it work on CI, we would need to integrate a small svs or till to the repo, or download it during test execution. Also, we would need to start TileServer on CI, that's doable too. @creilly8 @xzhou82

File: client/plots/wsiviewer/test/wsiviewer.integration.spec.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
+import * as helpers from '../../../test/front.helpers.js'
```
Comment (xzhou82) [inline client/plots/wsiviewer/test/wsiviewer.integration.spec.ts:1]: may remove `integration` from file name

File: client/plots/wsiviewer/test/wsiviewer.integration.spec.ts (hunk 3 above)
Comment (xzhou82) [reply client/plots/wsiviewer/test/wsiviewer.integration.spec.ts:27]: please do not enable it in CI for now

File: client/plots/sampleView.js [hunk 5]
```diff
@@ -554,19 +554,8 @@ class SampleView {
...
-					}
-				})
-
-				if (data.sampleWSImages?.length > 0) {
-					const cellDiv = div.append('div').style('display', 'inline-block')
-					this.wsiPlots.push({ sample, cellDiv })
-					wsiViewer(state.vocab.dslabel, cellDiv, this.app.opts.genome, sample.sampleName, data.sampleWSImages)
-				}
+				const cellDiv = div.append('div').style('display', 'inline-block')
+				wsiViewer(state.vocab.dslabel, cellDiv, this.app.opts.genome, sample.sampleName)
```
Comment (xzhou82) [inline client/plots/sampleView.js:558]: both dziviewer and wsiviewer should be dynamically imported

File: client/plots/wsiviewer/WSIViewer.ts (hunk 2 above)
Comment (aacic) [reply client/plots/wsiviewer/WSIViewer.ts:16]: And also how about to rename images.ts to wsimagesloaded.ts.

File: client/plots/wsiviewer/test/wsiviewer.integration.spec.ts (hunk 3 above)
Comment (aacic) [reply client/plots/wsiviewer/test/wsiviewer.integration.spec.ts:27]: Ok. It wouldn't be difficult to integrate it in the next steps.

File: client/plots/wsiviewer/WSIViewer.ts (hunk 2 above)
Comment (creilly8) [reply client/plots/wsiviewer/WSIViewer.ts]: @aacic, The file name is changed to `wsimagesloaded.ts`

File: client/plots/wsiviewer/defaults.ts (hunk 1 above)
Comment (creilly8) [reply client/plots/wsiviewer/defaults.ts]: @aacic, I reverted this change.

File: client/plots/wsiviewer/test/wsiviewer.integration.spec.ts (hunk 4 above)
Comment (creilly8) [reply client/plots/wsiviewer/test/wsiviewer.integration.spec.ts]: @xzhou82, No problem. File is renamed

File: client/plots/sampleView.js (hunk 5 above)
Comment (creilly8) [reply client/plots/sampleView.js:558]: @xzhou82, I believe we planned that out for the next PR. That's in step two here https://github.com/stjude/proteinpaint/issues/2021.

**Review by aacic — APPROVED** [review]

**Pull request merged**